spec/mail/fields/common/common_field_spec.rb

```diff
@@ -69,17 +69,8 @@
 
   end
 
-  context "when including the field name" do
-    it "does not strip out content that looks identitcal to the field name" do
-      field = Mail::SubjectField.new("Subject: Subject: for your approval")
-      expect(field.decoded).to eq("Subject: for your approval")
-    end
-  end
-
-  context "when not including the field name" do
```

Comment:
Thanks! It's to prove that this bug is still fixed: https://github.com/mikel/mail/pull/719/files